### ISSUE TITLE
Adding link to Emojipedia.org

### ISSUE THREAD

**Issue by rugk** [johannhof/emoji-helper#56]
I think a nice to have feature would be to have a link to http://emojipedia.org/ in the GUI somewhere when a emoticon is selected. This way you can easily check what is meant by this emoticon or how it looks with different emoticon styles.

**Comment by johannhof** [comment]
Yeah why not, if someone wants to take a stab at this feel free to.
